internal/credit/postgres_connector/test_helpers.go

```diff
@@ -0,0 +1,44 @@
+package postgres_connector
```

Comment:
Can we name it helpers_test.go or something, so that it's not available in normal builds only in tests?

Reply:
moved it to a dedicated package